Source/Core/Core/Boot/Boot_DOL.h

```diff
@@ -42,15 +44,16 @@ class CDolLoader
 		u32 bssAddress;
 		u32 bssSize;
 		u32 entryPoint;
-		u32 padd[7];
+		//u32 padding[7];
```

Comment:
Why's this still here? If it's commented out for informational purposes, it might want to be stated here.